numba/tests/test_np_functions.py

```diff
@@ -2503,6 +2507,43 @@ def check_pass_through(jitted, expect_same, params):
                 else:
                     check_pass_through(cfunc, True, params)
 
+    def test_repeat(self):
+        pyfunc = np_repeat
+        cfunc = jit(nopython=True)(pyfunc)
+
+        def check(a, repeats):
+            np.testing.assert_equal(pyfunc(a, repeats), cfunc(a, repeats))
```

Comment:
When checking results, use `self.assertPreciseEqual()` as it checks `dtype`, `layout`, `strides`, and various flags/properties are all equal too. See: https://github.com/numba/numba/blob/04c37092e20534a5794cb3dbfaae5dd69bcdc1cc/numba/tests/support.py#L219